hplsql/src/main/java/org/apache/hive/hplsql/ColumnMap.java

```diff
@@ -28,13 +28,25 @@ public class ColumnMap {
   private List<Column> columns = new ArrayList<>();
   private Map<String, Column> columnMap = new HashMap<>();
 
+  private Map<String, String> columnNameVsFQColumnName = new HashMap<>();
+
   public void add(Column column) {
     columns.add(column);
-    columnMap.put(column.getName().toUpperCase(), column);
+    String columnName = column.getName().toUpperCase();
+    columnMap.put(columnName, column);
+    if (columnName.contains(".")) {
```

Comment:
'.' is a valid character in a quoted identifier.
https://github.com/apache/hive/blob/e4348422c6f3b1910a8600ea7c7bd839894dcd6f/standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/utils/MetaStoreUtils.java#L131-L137
(above characters are accepted in any identifier not just in table names)

Can we have a separate field for table/alias name in the `Column` class?